message/message.go

```diff
@@ -2,6 +2,7 @@ package message
 
 type Message struct {
 	ResponseAPIEndpoint string  `json:"response_api_endpoint"`
+	PayloadType         string  `json:"payload_type"`
```

Comment:
Shouldn't this have `omitempty` since the default will be `tide` and we haven't implemented it in the SQS messages sent by the Sync Server?

Reply:
I'd rather it not be. It will be unmarshalled into an empty string if its not present and marshalled into an empty json string.  Eventually we want this to be defined. I think a "tide" fallback should be deprecated eventually and we need to be explicit.